tools/orca/profile-sample.toml

```diff
@@ -1,10 +1,19 @@
-control_uri = "https://localhost:8443"
+person_count_by_group = [
+  ["role_people_self_write_mail", 500]
+  ["role_people_self_set_password", 0],
```

Comment:
I've been thinking about this and I realised what's "off" here is that it's *super* unclear how it actually works when you look at it.

I think this should be:

```

[group."role_people_self_write_mail"]
member_count = 500

```

Then any *unspecified* group is just set to the default.

This can be done in the serde struct for toml with:

```
group: BTreeMap<String, GroupProperties>,

struct GroupProperties {
    member_count: Option<u32>
}
```

I think once you do this, it'll make the *rest* of the PR make a lot more sense, and be far clearer in how it reads.

Reply:
Yeah I should have done my own research into the toml crate before doing random stuff. I've fixed it now! (along with some extra *tiny* additions which you can see below :D )